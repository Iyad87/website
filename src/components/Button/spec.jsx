import React from 'react';
import { shallow } from 'enzyme';
import Button from './index';

test('components/Button snapshot', () => {
  const wrapper = shallow(<Button />);

  expect(wrapper).toMatchSnapshot();
});
