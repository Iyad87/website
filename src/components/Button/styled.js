import styled from 'styled-components';
import { highlight, textOnDark } from '../../styles/colours';

export const withButtonStyles = Component => styled(Component)`
  background: ${highlight};
  box-shadow: 0 25px 30px 5px rgba(88, 86, 214, 0.2);
  color: ${textOnDark};
  display: inline-block;
  font-size: 1.125em;
  font-weight: 400;
  text-align: center;
  text-decoration: none;
  min-width: 10em;
  padding: 20px;
`;
