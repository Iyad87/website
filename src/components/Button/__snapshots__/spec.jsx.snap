// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Button snapshot 1`] = `<styled.a />`;
